cmd/migrate/migrate.go

```diff
@@ -1,19 +1,63 @@
 package main
 
 import (
+	"strings"
+
 	"github.com/RedHatInsights/chrome-service-backend/rest/database"
 	"github.com/RedHatInsights/chrome-service-backend/rest/models"
 	"github.com/joho/godotenv"
 	"github.com/sirupsen/logrus"
+	"gorm.io/datatypes"
 	"gorm.io/gorm"
 )
 
+func getValidGridItems(items []models.GridItem) []models.GridItem {
+	newGridItems := []models.GridItem{}
+	for _, item := range items {
+		widgetTypes := strings.Split(item.ID, "#")
+		if len(widgetTypes) > 0 {
+			widgetType := widgetTypes[0]
+			err := models.AvailableWidgets(widgetType).IsValid()
+			if err == nil {
+				newGridItems = append(newGridItems, item)
+			} else {
+				logrus.Infof("Removing invalid widget type: %s", widgetType)
```

Comment:
@Hyperkid123 this looks good for now - though we may need to put some more thought into this long term. Such as handling situations where a widget type is renamed.

With this change all of the old widgets would be removed from the dashboards. 

I'm not implying that would happen often or ever (as the widgetType is just an identifier) - just that we should make communications clear in our readme about how those possible situations would be handled (or discouraging them from happening).